skyplane/_private/usage_constants.py

```diff
@@ -0,0 +1,32 @@
+USAGE_STATS_ENABLED_ENV_VAR = "SKYPLANE_USAGE_STATS_ENABLED"
```

Comment:
What is the purpose of putting this under _private? It's maybe better to put these files under a `skyplane.cli.usage` directory.

Reply:
`_private/` is meant for storing user private data. Ray does so [here](https://github.com/ray-project/ray/tree/b52cd964cb1d3696cb3c2bf43f5984bc49eb13e8/python/ray/_private/usage). But I can also move these files to `skyplane.cli.usage`.